cincinnati/src/plugins/internal/graph_builder/openshift_secondary_metadata_parser/plugin.rs

```diff
@@ -509,21 +510,16 @@ mod tests {
                 .graph
         };
 
-        // Sort the graphs for easier readable diffs
-        let graph_expected_sorted = {
-            let mut graph =
-                serde_json::to_value(&graph_expected).context("Sorting expected graph")?;
-            commons::testing::sort_json_graph_by_version(&mut graph);
-            graph
-        };
-
-        let graph_result_sorted = {
-            let mut graph = serde_json::to_value(&graph_result).context("Sorting result graph")?;
-            commons::testing::sort_json_graph_by_version(&mut graph);
-            graph
-        };
-
-        pretty_assertions::assert_eq!(graph_result_sorted, graph_expected_sorted);
+        if let Err(e) = compare_graphs_verbose(
+            graph_expected,
+            graph_result,
+            &[
+                "io.openshift.upgrades.graph.previous.remove",
+                "io.openshift.upgrades.graph.previous.remove_regex",
+            ],
+        ) {
+            panic!("{}", e);
```

Comment:
Not sure why we need to panic here - can't this func just return `Err`?

Reply:
This function is the test itself. Usually we use `unwrap()` also causes a panic, but it doesn't print the error message nicely, e.g. it doesn't even respect newlines but prints `\n` instead. `panic`ing here explicitly which respects the formatting, even color and such.

Reply:
Oh, I see, okay